### PR TITLE
fix: the output of command completion is displayed incorrectly

### PR DESCRIPTION
**Pull request by hung-cybo**
<!-- Thank you for sending a pull request! -->

## Why

Currently, the output of command completion is displayed incorrectly.
1) The alias does not align
2) The content is broken if there is a new line character in the description.

```
$ cli-kintone record import
--api-token                -- App's API token                                                                                                                                                                                                               
--app                      -- The ID of the app                                                                                                                                                                                                             
--attachments-dir          -- Attachment file directory                                                                                                                                                                                                     
--base-url                 -- Kintone Base Url                                                                                                                                                                                                              
--basic-auth-password      -- Kintone Basic Auth Password                                                                                                                                                                                                   
--basic-auth-username      -- Kintone Basic Auth Username                                                                                                                                                                                                   
--encoding                 -- Character encoding                                                                                                                                                                                                            
--fields                   -- The fields to be imported in comma-separated                                                                                                                                                                                  
--file-path                -- The path to source file.                                                                                                                                                                                                      
--guest-space-id           -- The ID of guest space                                                                                                                                                                                                         
--help                     -- Show help                                                                                                                                                                                                                     
--password                 -- Kintone Password                                                                                                                                                                                                              
--pfx-file-password        -- The password of client certificate file                                                                                                                                                                                       
--pfx-file-path            -- The path to client certificate file                                                                                                                                                                                           
--proxy                    -- The URL of a proxy server                                                                                                                                                                                                     
--update-key               -- The key to Bulk Update                                                                                                                                                                                                        
--username                 -- Kintone Username                                                                                                                                                                                                              
--version                  -- Show version number                                                                                                                                                                                                           
-p                     -u  --                                                                                                                                                                                                                               
The file extension should be ".csv"
```

## What

- [x] Do not display the alias (There is a [bug](https://github.com/yargs/yargs/issues/2268) of yargs)
- [x] Remove the line break in the description of the option
<!-- What is a solution you want to add? -->

## How to test

<!-- How can we test this pull request? -->
```
yarn build

# Config cli-kintone command completion. Ref: https://github.com/kintone/cli-kintone#command-completion

./cli.js record import --[Tab]
```

## Checklist

- [x] Read [CONTRIBUTING.md](https://github.com/kintone/cli-kintone/blob/main/CONTRIBUTING.md)
- [ ] Updated documentation if it is required.
- [ ] Added tests if it is required.
- [x] Passed `yarn lint` and `yarn test` on the root directory.

### CODE REVIEW

File: src/cli/record/export.ts [hunk 1]
```diff
@@ -20,21 +20,21 @@ const builder = (args: yargs.Argv) =>
...
     })
     .option("username", {
-      alias: "u",
       describe: "Kintone Username",
       default: process.env.KINTONE_USERNAME,
       defaultDescription: "KINTONE_USERNAME",
       type: "string",
       requiresArg: true,
     })
+    .alias("u", "username")
```
Comment (mshrtsr) [inline src/cli/record/export.ts:29]: I think it's better to add `TODO:` comments to WA.
e.g. `TODO: remove this workaround after https://github.com/yargs/yargs/issues/2268 is fixed.`

File: src/cli/record/import.ts [hunk 2]
```diff
@@ -75,7 +75,7 @@ const builder = (args: yargs.Argv) =>
       requiresArg: true,
     })
     .option("file-path", {
-      describe: 'The path to source file.\nThe file extension should be ".csv"',
+      describe: 'The path to source file. The file extension should be ".csv"',
```
Comment (mshrtsr) [inline src/cli/record/import.ts:78]: I think you can try the custom completion function.
https://yargs.js.org/docs/#api-reference-completioncmd-description-fn

The `.completion()` accepts a custom function before outputting the completion script.
You can escape line breaks inside it without removing the line break from the description.

Comment (mshrtsr) [reply src/cli/record/import.ts:78]: (Possibly, you can resolve the alias problem with this too.)

**Review by tasshi-me — CHANGES_REQUESTED** [review]

**Review by tasshi-me — CHANGES_REQUESTED** [review]
Works well!
But it's good to do a little refactoring.

```shell
$ node ./cli.js --get-yargs-completions record import -
--version:Show version number
--help:Show help
--base-url:Kintone Base Url
--username:Kintone Username
--password:Kintone Password
--api-token:App's API token
--basic-auth-username:Kintone Basic Auth Username
--basic-auth-password:Kintone Basic Auth Password
--app:The ID of the app
--guest-space-id:The ID of guest space
--attachments-dir:Attachment file directory
--file-path:The path to source file. The file extension should be ".csv"
--encoding:Character encoding
--update-key:The key to Bulk Update
--fields:The fields to be imported in comma-separated
--pfx-file-path:The path to client certificate file
--pfx-file-password:The password of client certificate file
--proxy:The URL of a proxy server
```

**Review by tasshi-me — APPROVED** [review]
LGTM!!

**Pull request merged**